src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_athena.json

```diff
@@ -0,0 +1,18 @@
+[
+ {
+  "op": "add",
+  "path": "/ValueTypes/AWS::Athena::CapacityReservation.Name",
+  "value": {
+   "StringMax": 128,
```

Comment:
Covered by regex:
```
"AWS::Athena::CapacityReservation.Name": {
   "AllowedPatternRegex": "[a-zA-Z0-9._-]{1,128}"
  },
```

Reply:
So when the regexp covers it, it overrides the need for any explicit `StringMax`/`StringMin`? In these case I checked the documentation and filled these in based on my reading of the regexp.